docs/content/create-medusa-app.mdx

```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
 
 ## Overview
 
-Medusa is composed of three different components: the headless backend, the storefront, and the admin dashboard.
+Medusa provides developers with the toolkit to create a commerce application including a Node.js backend that encapsulates the core package and the commerce modules, an admin dashboard, and a storefront. Developers can set up and use these tools separately, giving them a full ownership over their tech stack.
 
-Medusa provides the necessary tools and resources to set up the three components separately. This ensures that developers have full freedom to choose their tech stack, as they can choose any framework for the storefront and admin dashboard.
+In certain cases, it can make more sense to setup all the main building blocks together, such as when you're creating a fully fledged ecommerce store, or if you're curious about how Medusa works and what features it offers. For these cases, you can use the `create-medusa-app` command.
```

Comment:
todo: I think we can leave out this paragraph, as you can use `create-medusa-app` regardless of how you want to build your applications.